setup.py

```diff
@@ -34,7 +34,8 @@
         "construct-typing==0.5.4",
         "wxPython>=4.1.1",
         "arrow>=1.0.0",
-        "wrapt>=1.14.0"
+        "wrapt>=1.14.0",
+        "typing_extensions>=4.4.0"
```

Comment:
on pypi it is called typing-extensions (minus instead of underscore)

Reply:
Perfect, works